cookbook/core/intermediate/raw_container.py

```diff
@@ -26,7 +26,7 @@
     output_data_dir="/var/outputs",
     inputs=kwtypes(val=int),
     outputs=kwtypes(out=int),
-    image="alpine",
+    image="public.ecr.aws/a1p3q7r0/alpine:3.12",
```

Comment:
so the old ones downloads from docker hub... this downloads from ECR... which one is the officially maintained version? we should stick with that...